### PR TITLE
Trader, TraderAction, TraderPeerMsg, PeerMsg completed, Peer Still Working On

### PR DESCRIPTION
**Pull request by Victoriatheeast**
I don't wanna mess up what I did, so I just created a new branch.
- For the market's local database, would you like to connect to the sqlite or just scanning files like I did. 
- For the Trader, I scan the trading request for my own testing purpose.
- For the two-phase commit, is it just inquire price first and then sends along the request?
- For notifying the admin server about the suspection of a peer, what I can do is to create a concurrent hash map, as long as someone leaves or joins the cluster, check if its logical name is not super peer, that's what I have thought of so far.

Hopefully I can push codes for updating Peer and SuperPeer before later tonight, so that you can take a look at it.

### CODE REVIEW

**Comment by meyermt** [comment]
@Victoriatheeast , are you saying we should try to merge this pull request?

**Comment by meyermt** [comment]
Looks like it has no conflicts, which is good 👍

File: .gitignore [hunk 1]
```diff
@@ -81,5 +81,5 @@ fabric.properties
 ./**/*.db
 
 # price file
-*.csv
+#*.csv
```
Comment (meyermt) [inline .gitignore]: I don't think we need/want to version the csv files, right?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Tue May 02 18:13:36 CDT 2017
+#Fri Jun 02 11:05:27 CDT 2017
```
Comment (meyermt) [inline gradle/wrapper/gradle-wrapper.properties]: I'm going to remove this change if you are ok with that.

Comment (meyermt) [inline gradle/wrapper/gradle-wrapper.properties]: Also going to remove your version of the gradle wrapper. Shouldn't need to update that.

File: shared/src/main/java/com/vam/json/Stock.java [hunk 3]
```diff
@@ -1,47 +1,145 @@
...
+ * The stock class renders and updates the stock name, value, and the number of shares available
  */
+
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+
 public class Stock {
```
Comment (meyermt) [inline shared/src/main/java/com/vam/json/Stock.java]: additions to the Stock class make sense 😄

File: .gitignore (hunk 1 above)
Comment (Victoriatheeast) [inline .gitignore]: This is for test purpose though, so I actually created my own.

**Comment by Victoriatheeast** [comment]
@meyermt , yes, sounds good.

**Pull request merged**